conf/nginx/server.tpl.conf

```diff
@@ -15,16 +15,9 @@ server {
     location ^~ '/.well-known/ynh-diagnosis/' {
         alias /var/www/.well-known/ynh-diagnosis/;
     }
-    {%- if mail_enabled == "True" %}
-
-    location ^~ '/.well-known/autoconfig/mail/' {
-        alias /var/www/.well-known/{{ domain }}/autoconfig/mail/;
-    }
-    {%- endif %}
```

Comment:
\<whisper\>not sure why this got deleted :eyes: \</whisper\>

Reply:
heh I guess it shouldn't be accessible by unencrypted plain text http and redirect to https instead :shrug:

Reply:
Ah yes I didnt realize it's for HTTP hmmmm I wonder why it's still there, I sort of remember reworking and pondering many time what we leave in the plain HTTP part ... there might be a boring story of "actually mail client may test the HTTP first and then 'fallback'(?)(sigh) to HTTPS ... I hope since then mail client do check HTTPS first and anyway yeah it should be ok to throw a 302 :white_check_mark: